cmake/developer_package/cross_compile/python_helpers.cmake

```diff
@@ -76,7 +76,7 @@ macro(ov_find_python3 find_package_mode)
         ov_cross_compile_define_debian_arch()
     endif()
 
-    find_package(Python3 ${find_package_mode} COMPONENTS Interpreter ${python3_development_component})
+    find_package(Python ${find_package_mode} COMPONENTS Interpreter ${python3_development_component})
```

Comment:
how does it help?
We don't need to find Python 2.x

Reply:
The problem was in GenAI reading Python_VERSION_MAJOR instead of Python3_VERSION_MAJOR